Esteban-Salas-Posada/tarea-3/last-survivor.js

```diff
@@ -0,0 +1,11 @@
+function lastSurvivor(letters, coords) {
+    var newArray;
+    coords.forEach((value,index) => {
+      newArray = letters.split('');
```

Comment:
@esteban-salas , this could be one line

```suggestion
      letters.split('').splice(value, 1).join('')
```